### PR TITLE
authors: fix sort

### PR DESCRIPTION
**Pull request by harunurhan**
* Adds bestmatch sort option and makes it default for author search.
* Also adds docs for efficiently working with visual e2e tests and docker.

Ref: #1279

### CODE REVIEW

File: backend/inspirehep/search/mappings/v7/records/authors.json [hunk 1]
```diff
@@ -274,7 +277,8 @@
             "copy_to": [
               "authorautocomplete",
               "names_analyzed_initials",
-              "names_analyzed"
+              "names_analyzed",
+              "name_keyword"
```
Comment (michamos) [inline backend/inspirehep/search/mappings/v7/records/authors.json:281]: I don't think that's a correct use of `copy_to`. It's meant when you want to combine several fields into one. Better define a `value.raw` of type `keyword` as a variant, like we do for example here: https://github.com/inspirehep/inspirehep/blob/95af58d4b474a36dc9b0a1f37560d831fe793106/backend/inspirehep/search/mappings/v7/records/hep.json#L200-L204

Comment (harunurhan) [reply backend/inspirehep/search/mappings/v7/records/authors.json:281]: I guess it's also better to have `lowercase_normalizer` just incase?

**Review by michamos — APPROVED** [review]

**Comment by harunurhan** [comment]
We should remember to remove the hotfix on prod, when this goes in @michamos

**Comment by michamos** [comment]
You can already do it on master so it goes to QA for testing, and it will be deployed on prod when we do the next release of hep.

**Pull request merged**